<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="excelFilenameLineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="databaseFileLineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="scriptPathLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="chooseExcelFileBtn">
         <property name="text">
          <string>选择Excel文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseDatabaseFileBtn">
         <property name="text">
          <string>选择Sqlite文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseScriptPathBtn">
         <property name="text">
          <string>选择Script保存路径</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="generateBtn">
     <property name="text">
      <string>开始生成</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="infoListWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="clearBtn">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
